cwe_api/cvss_inferencer.py

```
@@ -0,0 +1,145 @@
+        self.classifiers = nn.ModuleDict({
+            output: nn.Linear(self.bert.config.hidden_size, num_labels[output]) for output in OUTPUTS_NAME
+        })
+
+    def forward(self, input_ids, attention_mask):
+        bert_outputs = self.bert(input_ids=input_ids, attention_mask=attention_mask)
+        cls_output = bert_outputs.pooler_output  # Get the CLS token output
+        cls_output = self.dropout(cls_output)
+        
+        # Apply classifier for each output
+        output_logits = {output: classifier(cls_output) for output, classifier in self.classifiers.items()}
+        return output_logits
```

Ruff

Unnecessary assignment to `output_logits` before `return` statement

Remove unnecessary assignment

(RET504)